plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveHadoop.java

```diff
@@ -101,17 +103,47 @@ public HostAndPort getHiveMetastoreEndpoint()
     public static class Builder
             extends BaseTestContainer.Builder<HiveHadoop.Builder, HiveHadoop>
     {
+        private boolean disableHdfsAndHiveRuntime;
+
         private Builder()
         {
             this.image = DEFAULT_IMAGE;
             this.hostName = HOST_NAME;
             this.exposePorts = ImmutableSet.of(HIVE_METASTORE_PORT);
         }
 
+        public Builder disableHdfsAndHiveRuntime()
+        {
+            disableHdfsAndHiveRuntime = true;
+            return this;
+        }
+
         @Override
         public HiveHadoop build()
         {
-            return new HiveHadoop(image, hostName, exposePorts, filesToMount, envVars, network, startupRetryLimit);
+            HiveHadoop hiveHadoop = new HiveHadoop(image, hostName, exposePorts, filesToMount, envVars, network, startupRetryLimit);
```

Comment:
What about using a dedicated HMS docker image instead of stripping on the fly  the services?

https://hub.docker.com/r/starburstdata/hive-metastore may be a viable alternative.

Reply:
There is also the effort of @nineinchnick to get multi arch support on the existing images. See https://github.com/trinodb/docker-images/pull/143

Not all images include multi arch support but `hive3.1-hive` has, I guess we can use that image easily for the Iceberg and Delta tests.